openapi/trireg2.yaml

```diff
@@ -2,9 +2,33 @@ openapi: "3.0.3"
 info:
   title: "TriReg2 API"
   description: "An OpenAPI specification for time registrations"
-  version: "0.1.1"
+  version: "0.1.2"
+  contact:
+    name: "TBD"
+    email: "placeholder@trifork.com"
```

Comment:
I will create an issue to investigate which exact details to put in here

Reply:
https://github.com/trifork/trireg2-api/issues/36